README.md

```diff
@@ -1,2 +1,50 @@
-# research-frontend
-Public research frontend SPA for CA
+# Research Frontend for CollectiveAccess
+
+Public research frontend SPA for CollectiveAccess.
+
+## Development Environment Setup
+
+You need a recent version of Node.js installed.  For development environments, `nvm` is recommended: 
+https://github.com/creationix/nvm.  Then execute the following commands in the project's root directory:
+
+    npm install -g gulp bower
+    npm install
+    bower install
+
+This (1) Installs global dependencies, (2) installs local development and automation dependencies, and (3) installs 
+application dependencies.  You can check `package.json` and `bower.json` to see what will be installed.
+
+## Automation
+
+The project uses `gulp` to automate many processes in the development environment.
+
+### Performing QA
+
+To run linter, code style checker and unit tests:
+
+    gulp qa
```

Comment:
This is strange, I thought it was just an unsaved dependency but I've deleted `node_modules` and done `npm install` again, and it Works For Me (TM).  I can't see it listed globally either.

Can you please do `npm ls --depth=0` and post results?  

You could also try removing lines 17-18 of `gulpfile.js` just to confirm that everything else is working and the only problem is with the reporter.